src/node_worker.cc

```diff
@@ -407,8 +464,14 @@ void Worker::OnThreadStopped() {
                   env()->message_port_string(),
                   Undefined(env()->isolate())).FromJust();
 
-    Local<Value> code = Integer::New(env()->isolate(), exit_code_);
-    MakeCallback(env()->onexit_string(), 1, &code);
+    Local<Value> args[] = {
+      Integer::New(env()->isolate(), exit_code_),
+      custom_error_ != nullptr ?
+          OneByteString(env()->isolate(), custom_error_).As<Value>() :
+          Null(env()->isolate()).As<Value>(),
+    };
+
+    MakeCallback(env()->onexit_string(), arraysize(args), args);
```

Comment:
Maybe we can pass the limits back here as well to display them in the error messages?

Reply:
Do you have suggestions for how/when to pass them back? I don’t think you’re thinking about callback arguments, right?

Reply:
If we have those as AliasedBuffers we could pass them back here right? (Or is that not guaranteed to be valid at this point?)